packages/ember-template-compiler/lib/system/compile.ts

```diff
@@ -5,7 +5,11 @@ import require, { has } from 'require';
 import { CompileOptions } from './compile-options';
 import precompile from './precompile';
 
-let template: (templateJS: () => string) => string;
+// FIXME
```

Comment:
Is this a blocker for merging, or just a future refactor/todo?

Reply:
Future refactor/todo: make the types better than just "stubbing" it with `any`/`unknown` (pretty widespread issue in the codebase right now)